README.md

```diff
@@ -14,7 +14,7 @@
 
 ## Contribute
 
-There are many ways to **[contribute](https://github.com/bullhorn/career-portal/blob/master/CONTRIBUTING.md)** to Bullhorn Career Portal.
+There are many ways to **[contribute](https://gist.github.com/MarcDiethelm/7303312)** to Bullhorn Career Portal.
```

Comment:
I would not like to make this change as the audience for the contributing guide is not someone who is new to contributing on github, but one who is contributing to this specific repository.   Over the next few days I am fleshing out the contributing file.

Reply:
Copy that, makes sense @charlesabarnes.

Reply:
@charlesabarnes - I tried your suggestion regarding chaining the boolean conditions and when I do so, the end result is not desired. I changed my original PR code to the following:
```
appConfig.eeoc.veteran = (argv.eeocAll || argv.eeocVeteran || argv.eeocV);
appConfig.eeoc.genderRaceEthnicity = (argv.eeocAll || argv.eeocGenderRaceEthnicity || argv.eeocGRE);
appConfig.eeoc.disability = (argv.eeocAll || argv.eeocDisability || argv.eeocD);
```

And the dist/app.json file has this for the resulting `eeoc` section:
```
    "eeoc": {}
```

Reply:
Ah I see my mistake, its still being interpreted as a string.

Instead, can you try 

```
    appConfig.eeoc.genderRaceEthnicity = argv.eeocAll === 'true' || argv.eeocGenderRaceEthnicity === 'true' || argv.eeocGRE === 'true';
    appConfig.eeoc.veteran = argv.eeocAll === 'true' || argv.eeocVeteran === 'true' || argv.eeocV === 'true';
    appConfig.eeoc.disability = argv.eeocAll === 'true' || argv.eeocDisability === 'true' || argv.eeocD === 'true';
```

Reply:
@charlesabarnes - your proposed changes still do not work. They fix the issue of the object attributes being completely omited but the values set forth in the app.json.template file are not honored. I have all three eeoc attributes set to `true` in my app.json.template file and when I build with `gulp`, the dist/app.json has them set to false.